### PR TITLE
Put config key names in new ReactorConfigKeys class

### PR DESCRIPTION
**Pull request by twaugh**
This allows the default configuration to be specified using these keys.

Signed-off-by: Tim Waugh <twaugh@redhat.com>

### CODE REVIEW

**Comment by twaugh** [comment]
cc @csomh

File: atomic_reactor/plugins/pre_reactor_config.py [hunk 1]
```diff
@@ -50,26 +50,28 @@ def __init__(self, name, max_concurrent_builds, enabled=True):
         self.enabled = enabled
 
 
+class ReactorConfigKeys(object):
```
Comment (csomh) [inline atomic_reactor/plugins/pre_reactor_config.py]: I assume the intention is to be able to use this in other/upcoming plugins. In this case, maybe a one-line doc-string would be nice - what is this, and what is the intention (even though at this point this is pretty obvious).

Comment (twaugh) [inline atomic_reactor/plugins/pre_reactor_config.py]: Really it's for use by the ReactorConfig class, which is concerned with fetching values for specific keys in the configuration file.

Other modules are expected to use the ReactorConfig instance returned by get_config(), which can provide methods for making queries against the configuration file.

Comment (csomh) [inline atomic_reactor/plugins/pre_reactor_config.py]: I see, then it's fine this way, too.

**Pull request merged**